mne/io/base.py

```diff
@@ -20,7 +20,9 @@
 import numpy as np
 
 from .constants import FIFF
-from .utils import _construct_bids_filename, _check_orig_units
+from .utils import _construct_bids_filename, _check_orig_units, \
+    _get_cart_ch_coords_from_inst
+from ..utils.check import _infer_check_export_fmt
```

Comment:
Should be down in the `from ..utils` line, you'll need to add `_infer_check_export_fmt` to `mne/utils/__init__.py`